src/constants/mockData.js

```diff
@@ -0,0 +1,9 @@
+export const MOCK_TODO_LIST = [
```

Comment:
We can move this constant file closer to the file where it is consumed, Generally we move helpers/constants/utils which are needed in multiple modules to such a point where it can referred and reused. If that is not a case please keep things closer to the place where it is consumed.